papers/Vvmax/Data/all_unloc_frb_data.dat

```diff
@@ -0,0 +1,64 @@
+# FRB                 DM        Freq          S/N    S/Nth   Dmg    dt         w         theta_d     Fnu       Zmac Nants RF
```

Comment:
Is this data that others will want to use for future analyses?

if so, let's "expose" it by having it in `zdm/data/`

Reply:
OK, I'll have a think about that. Ideally, one might open a survey file and do V/Vmax for that survey. I don't think I'll implement that however. But certainly some routines would be useful. Good point, I'll check it out, and maybe re-factorise some routines.

Reply:
I've shifted some routines to zdm for vvmax - notably, the actual calculations of V and Vmax! - but the data is in a pretty unique format. I think I'll keeo it that way

Reply:
So see new file vvmax.py in the zdm directory